albumentations/random_utils.py

```diff
@@ -29,7 +29,13 @@ def uniform(
 def rand(d0: NumType, d1: NumType, *more, random_state: Optional[np.random.RandomState] = None, **kwargs) -> Any:
     if random_state is None:
         random_state = get_random_state()
-    return random_state.randn(d0, d1, *more, **kwargs)  # type: ignore
+    return random_state.rand(d0, d1, *more, **kwargs)
+
+
+def randn(d0: NumType, d1: NumType, *more, random_state: Optional[np.random.RandomState] = None, **kwargs) -> Any:
+    if random_state is None:
+        random_state = get_random_state()
+    return random_state.randn(d0, d1, *more, **kwargs)
```

Comment:
add `# type: ignore` after each return statement (as before) to make an exception for type checking in mypy

Reply:
Thanks, I added the statement.